kubernetes/prod/07-mongodb-statefulset.yaml

```
@@ -0,0 +1,215 @@
+            - "--bind_ip_all"
+            - "--replSet"
+            - "rs0"
+            - "--auth"
+            - "--wiredTigerCacheSizeGB"
+            - "0.25"
+          ports:
+            - containerPort: 27017
+              name: mongodb
+              protocol: TCP
+          env:
+            - name: MONGO_INITDB_ROOT_USERNAME
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)